packages/core/strapi/lib/commands/transfer/import.js

```diff
@@ -8,33 +8,59 @@ const {
   // eslint-disable-next-line import/no-unresolved, node/no-missing-require
 } = require('@strapi/data-transfer');
 
-module.exports = async () => {
-  console.log('Importing data...');
+const strapi = require('../../Strapi');
 
-  // From file
-  const source = createLocalFileSourceProvider({ backupFilePath: './backup.tar.gz' });
+const logger = console;
 
-  // To Strapi
-  const destination = createLocalStrapiDestinationProvider({
+module.exports = async (args, unknownArgs) => {
+  /**
+   * From strapi backup file
+   */
+
+  // treat any unknown arguments as filenames
+  if (unknownArgs.args.length !== 1) {
```

Comment:
For import I thought it was better to be explicit, otherwise we have to search the directory for a possible backup file and make a guess at what they want to import.

Reply:
Alright, let's require that for now. We can still add a default value later if it's really a need.